### DISCUSSION TITLE
Question in response to issue

### DISCUSSION THREAD

**Discussion by Naren219** [PWNAGERobotics/ScoutingPASS#32] [General]
Do you use special software or features to get a good representation (heat map) of where opponent are mainly shooting from?

**Comment by fuddster** [comment]
There are a few ways to do it. 
1) You can graph it in Excel, counting how many times a location shows up in your data.   The highest counts are where they like to shoot from the most.
2) Tableau is powerful too.  You can do some nice graphs with it there
3) We wrote a custom VBA script in Excel to populate a 12x6 grid with the # of times that spot shows up in the data.   Then we used  Conditional Formatting to vary the color from white to dark green depending on the frequency for each spot.  To finish it off we put the field image over those cells and made it mostly transparent so that it was easy to identify the most frequent shooting spots.

**Reply by Naren219** [reply]
Cool! Thank you so much! Is that VBA script public?

**Reply by fuddster** [reply]
We will add that to the Excel spreadsheet eventually.   We are really busy at competitions right now.

**Reply by Naren219** [reply]
Can you add the VBA script to Github?

**Reply by Naren219** [reply]
Can you add that script to GitHub? We are qualified for worlds and really need this type of visual data for the rest of the team.

**Reply by fuddster** [reply]
I've added another Excel spreadsheet to the repository under the Excel directory.   It has an example of how to do the shooting heatmaps in Excel.

**Reply by Naren219** [reply]
Thank you so much for these tools! You truly have great skill. Good luck on your match tomorrow (I'm in the TerrorBytes):)

**Reply by fuddster** [reply]
Good luck tomorrow!  

It's going to be an interesting alliance selection.  I hope ScoutingPASS is helping you tonight!
